src/python/README.md

```diff
@@ -8,27 +8,20 @@ cd Reference-NetApp
 python3 -m venv env
 source env/bin/activate
 
-cd
-cp -r /srv/opencv ~
-pip3 install ~/opencv/*.whl
+# Install interface from era-5g-interface repository
+pip3 install era-5g-interface
```

Comment:
Is this necessary? It should be in the requirements.txt with a correct version

Reply:
It is not necessary... you're right